wrapanapi/containers/__init__.py

```diff
@@ -13,9 +14,24 @@ class ContainersResourceBase(object):
     directed to the path of the resource.
     The following parameters should be statically defined:
         * RESOURCE_TYPE: (str) the resource type name in the API
+        * KIND: (str) the resource 'Kind' property as it appear in JSON
+        * CREATABLE (optional): (bool) Specify whether this resource is creatable or not (some
+                                resources are not, e.g. Pod is created by Replication Controller
+                                and not manually). set to False by default.
+        * API (optional): (str) The API to use - the default is Kubernetes ('k_api') but some
+                          resources use OpenShift ('o_api').
         * (optional) VALID_NAME_PATTERN: (str) the regex pattern that match a valid object name
     """
+    CREATABLE = False
+    API = 'k_api'
```

Comment:
I propose that we use an importable CONST here to help prevent string errors, as they will be detected at import time.